packages/yoga/src/Input/web/Input.jsx

```diff
@@ -78,6 +78,13 @@ const IconWrapper = styled.div`
   `}
 `;
 
+const LeftElementWrapper = styled.div`
+  position: absolute;
+  left: 8px;
```

Comment:
Check if it'd better to use yoga spacing props

Reply:
Thanks for the tip. Refactored to use the _spacing_ prop with the _xxsmall_ variation.